src/aws_iot_shadow_json.c

```diff
@@ -424,13 +462,13 @@ bool isReceivedJsonValid(const char *pJsonDocument) {
 	return true;
 }
 
-bool extractClientToken(const char *pJsonDocument, char *pExtractedClientToken) {
+bool extractClientToken(const char *pJsonDocument, size_t jsonSize, char *pExtractedClientToken, size_t clientTokenSize) {
 	int32_t tokenCount, i;
 	uint8_t length;
```

Comment:
Are you sure `uint8_t` (up to 255) is always sufficient?  How about `size_t` instead?

Reply:
Fixed

Reply:
It's still `uint8_t` in the latest version?